setup.sh

```diff
@@ -37,7 +37,11 @@ function updateenv () {
 function install_talib () {
     curl -O -L http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
     tar zxvf ta-lib-0.4.0-src.tar.gz
-    cd ta-lib && ./configure --prefix=/usr && make && sudo make install
+    cd ta-lib
+    sed -i.bak "s|0.00000001|0.000000000000000001 |g" src/ta_func/ta_utility.h
+    ./configure --prefix=/usr/local
+    make
+    sudo make install
     cd .. && rm -rf ./ta-lib*
```

Comment:
i think this removes the file in our repository ... leading to a semi-dirty git state.

`rm -rf ./ta-lib/` should leave the downloaded file in peace - while still removing the directory.